RecoTauTag/RecoTau/python/RecoTauPiZeroBuilderPlugins_cfi.py

```diff
@@ -65,19 +61,15 @@
     minStripEt = cms.double(1.0),
     updateStripAfterEachDaughter = cms.bool(False),
     maxStripBuildIterations = cms.int32(-1),
-    verbosity = cms.int32(0)
 )
 
 # Produce a "strips" of photons
 # with no track quality cuts applied to PFElectrons
 # and eta x phi size of strip increasing for low pT photons
 
-modStrips2 = cms.PSet(
-    name = cms.string("s"),
+modStrips2 = strips.clone(
     plugin = cms.string('RecoTauPiZeroStripPlugin3'),
     applyElecTrackQcuts = cms.bool(False),
```

Comment:
just FYI: this will conflict with https://github.com/cms-sw/cmssw/pull/22594/files#diff-c6aeafdc9967cd7d3a09834a42822a99R78

Reply:
Yes, that's the only conflict, as far as I can see.
Well, maybe not even a conflict, because github may realize that the updates are in different parts of the same file.
In any case, the last addition  https://github.com/cms-sw/cmssw/pull/22594/files#diff-c6aeafdc9967cd7d3a09834a42822a99R78 will not be needed any more once this PR is merged, and I would simply ask to revert the last commit (or simply remove that line) from #22594

Reply:
the context lines overlap; so, it will trigger a conflict.